front/app/containers/Admin/formBuilder/components/SortableRow/FlexibleRow/index.tsx

```diff
@@ -0,0 +1,89 @@
+// Libraries
+import React, { ReactNode } from 'react';
+
+// Style
+import styled from 'styled-components';
+import CSSTransition from 'react-transition-group/CSSTransition';
+import { fontSizes, colors } from 'utils/styleUtils';
+import { Box } from '@citizenlab/cl2-component-library';
+
+const timeout = 200;
+
+export const Container = styled.div`
```

Comment:
All of this CSS is from the original SortableRow component. Since it's transitions/dynamic we need to use a styled component. I wanted to make sure it's as similar to the original component as possible, but maybe there are some improvements we can make to clean it up later on?